data/governments.txt

```diff
@@ -398,3 +414,10 @@ government "Wanderer"
 	"friendly disabled hail" "wanderer untranslated"
 	"hostile hail" "wanderer untranslated"
 	"hostile disabled hail" "wanderer untranslated"
+
+# A dummy government used to prevent Hai from entering the wormhole in Waypoint/Ultima Thule, but allowing human merchants and other governments to enter.
+government "Wormhole Alpha"
+	color .4 .4 .4
+	"attitude toward"
+		"Hai" -.3
+		"Hai (Unfettered)" -.3
```

Comment:
Set these values to -.01 so that the player's rep with Wormhole Alpha doesn't change; repairing Hai could lead to the player being unable to access the wormhole under this implementation.

Reply:
So many small details I don't know, anything else?